packages/devextreme/js/ui/scheduler.d.ts

```diff
@@ -78,6 +78,8 @@ export type CellAppointmentsLimit = 'auto' | 'unlimited';
 /** @public */
 export type RecurrenceEditMode = 'dialog' | 'occurrence' | 'series';
 /** @public */
+export type SchedulerAppointmentFormIconsMode = 'both' | 'main' | 'recurrence' | 'none';
```

Comment:
I don't really like the name of the type. I suggest:
```suggestion
export type SchedulerAppointmentFormIconDisplay = 'both' | 'main' | 'recurrence' | 'none';
```